src/Silakov.Homework4/IMenu.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Silakov.Homework4
+{
+    interface IMenu
```

Comment:
please try to implement a solution showing the purpose of interface, the syntax is the only valuable thing here